svgscreen/svgscreen.go

```diff
@@ -320,7 +324,7 @@ func (s *Screen) Render(w io.Writer) error {
 	} else {
 		// Pixel coordinates
 		w := float32(s.CharacterBoxSize.X*s.TerminalWidth) + 2*s.MarginSize.X
-		h := float32(s.CharacterBoxSize.Y*s.NrLines) + 2*s.MarginSize.Y
+		h := float32(s.CharacterBoxSize.Y*s.NrLines)*s.LineHeight + 2*s.MarginSize.Y
```

Comment:
I guess this makes lineheight-grid-charbox.svg test have weird height? just skip `*s.LineHeight`? 🤔

Reply:
yeh, maybe skip on grid mode?

Reply:
I think maybe grid and "em"-mode might make sense also? I pushed fix to ignore line height in charbox mode and also made the tests a bit more exaggerated to be easier to spot problems